BBKoffieTuin/Assets/Scripts/PhoneDirection.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using TMPro;
+using UnityEngine;
+using UnityEngine.Events;
+
+public class PhoneDirection : MonoBehaviour
+{
+    [SerializeField] private float rotationTolerance = 1f;
+    
+    private Vector2 _currentPos;
+    private Vector2 _nextPos = new Vector2(4.856711f, 52.390652f);
```

Comment:
Harde default values? is dit nodig zo niet is een vector2.zero niet beter voor readability.

Reply:
Nee, dit is een positie bij MediaCollage zelf wat testen makkelijker maakt